modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/insert/nested-repeater-create-container.js

```diff
@@ -16,51 +17,25 @@ export class NestedRepeaterCreateContainer extends Base {
 		return 'document/repeater/insert';
 	}
 
-	apply( args ) {
-		const { containers = [ args.container ] } = args,
-			component = $e.components.get( 'nested-elements/nested-repeater' );
+	getConditions( args ) {
+		// Will only handle when command called directly and not through another command like `duplicate` or `move`.
+		const isCommandCalledDirectly = $e.commands.isCurrentFirstTrace( 'document/repeater/insert' );
```

Comment:
maybe use `this.getCommand()` for the parameter?

Reply:
Done